Makefile

```diff
@@ -35,6 +37,7 @@ BENCHTIMEOUT := 5m
 TESTFLAGS    :=
 STRESSFLAGS  :=
 DUPLFLAGS    := -t 100
+STARTFLAGS   :=
```

Comment:
a reasonable default for this might be `-s type=mem,size=1GiB --alsologtostderr`